index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Website</title>
+    <link rel="stylesheet" href="./styles.css">
+    <link rel="icon" href="./favicon.ico" type="image/x-icon">
+  </head>
+  <body>
+
+  <!-- HEADER SECTION (major section) -->
+
+  <header id="headerSection">
+
+  <!-- navigation area (subsection) -->
+    <nav id="navBar">
+      <div id="logoContainer" class="navitems">
+        <p id="logo">My Logo</p>
+      </div>
+
+      <div id="menuiconContainer" class="navItems">
+      	<img src="./media-library/Union.png" alt="menu" width="" height="">
+      </div>
+
+    </nav>
```

Comment:
- Kindly make sure that the container of the navigation barre section is in the `header` tag. :+1: 

* Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.The container of the section is a header tag.


- Kindly make sure the logo (if the design has one) has a link (`a` tag). :+1: 

   * This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
   
- and also the hamburger button has a `nav` tag.

   * The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices. The logo (if the design has one) is on the left and has a link (a tag).

```
eg.   
            
<header id="navBar">
      <div id="logoContainer" class="navitems">
      <a href="#"> My Logo</a>
      </div>

      <nav id="menuiconContainer" class="navItems">
      	<img src="./media-library/Union.png" alt="menu" width="" height="">
      </nav>

    </header>

```